src/Microsoft.AspNetCore.Mvc.TagHelpers/InputTagHelper.cs

```diff
@@ -230,6 +230,33 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
             }
         }
 
+        /// <summary>
+        /// Gets an &lt;input&gt; elements "type" attribute value for the given <paramref name="modelExplorer"/>.
+        /// </summary>
+        /// <param name="modelExplorer">The <see cref="ModelExplorer"/> to inspect.</param>
+        /// <param name="inputTypeHint">When this method returns, contains a type hint associated with the given
+        /// <paramref name="modelExplorer"/>.</param>
```

Comment:
Haven't heard from @marcusien so I'm assuming more info is better than less here.